spec/system/admin_views_case_contact_index_spec.rb

```diff
@@ -1,25 +0,0 @@
-require "rails_helper"
```

Comment:
originally, I tried to copy this test and use in in a new /system/volunteer_views_a_casa_case_spec.rb file (see the first commit #3f35d3b).

unfortunately, I tried removing the [corresponding style](https://github.com/rubyforgood/casa/blob/50d0c803a277dcf0af6d227ecd8d266c92274981/app/javascript/src/stylesheets/pages/case_contacts.scss#L30-L33) and this test continued to succeed, so I think we can remove it?